cli/cmd/plugin/unmanaged-cluster/config/config.go

```diff
@@ -16,6 +16,8 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/kubeconfig"
```

Comment:
Local imports should be after third part imports, so this should move under `gopkg.in`.

Reply:
Hmmm is that something `go fmt ./...` won't correct for?

Reply:
I don't think this is something that `go fmt` can handle, but there is a `--fix` option for golangci-lint that does.